var/spack/repos/builtin/packages/hypre/package.py

```diff
@@ -175,6 +181,11 @@ def _configure_args(self):
                 '--enable-curand',
                 '--enable-cub'
             ])
+            # New in 2.21.0: replaces --enable-cub
```

Comment:
There was a typo in the title (`2.21.1` rather than `2.22.1` so I wanted to check if in fact in this case you meant `@2.21.0:`

Reply:
Thanks for correcting the typo. The comment applies to 2.21.0 so it's OK.